### DISCUSSION TITLE
[BUG] out of ideas...

### DISCUSSION THREAD

**Discussion by NipBoss** [Chia-Network/chia-blockchain#3438] [Support]
**Describe the bug**
No ability to plot, error is possibly related to permissions

**To Reproduce**
See screenshots, all permissions are granted, all drives available, shared, and given all access privileges to the admin user.

**Expected behavior**
Does not allow any plots to populate, gets stuck and erroed out at 1%, please see error log.
Tried with same python v3.7, reinstalled, no help
Tried with pyhon v3.9.4, reinstalled, no help, same issue

**Screenshots**
https://imgur.com/a/X8UzPyj

**Desktop (please complete the following information):**

- OS: Windows 10 Pro
- OS Version/Flavor: 20H2
- CPU: AMD Ryzen 7 3700X 8-Core Processor                3.60 GHz
- Storage: 170TB (Seagate x 10TB)
- GPU: AMD Radeon R9 200 Series

**Additional context**
Tried to reinstall OS, reinstall Chia, reinstall dependencies, re-grant full read/write to all drives to all users,
still does not sync and errors out. Please help. Error log is attached.

[log.txt](https://github.com/Chia-Network/chia-blockchain/files/6411670/log.txt)

**Comment by NipBoss** [comment]
Hello guys, I would love to be ready at the network launch tomorrow, I am synched and cannot get any plots going :(
Anyone out there has any idea how to fix this issue?

**Comment by NipBoss** [comment]
Can someone help out? :(

**Reply by bdelgado1995** [reply]
It looks like it's trying to plot directly into c:\ which Windows may not allow. Can you try making c:\plots and try to plot to that? Also you may not want to plot on your OS drive. What drive letter(s) are your Seagates and can you point them there?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
